std_msgs/Header header
string camera_name
string vehicle_name
float64 roll
float64 pitch
float64 yaw